### PR TITLE
Update News.md

### PR DESCRIPTION
**Pull request by ritikakundra**
Fix # (see https://help.github.com/en/articles/closing-issues-using-keywords)

Describe changes proposed in this pull request:
- a
- b

# Checks
- [ ] Runs on heroku
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). It can help to figure out who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them either through slack or by assigning them as a reviewer on the PR

### CODE REVIEW

File: docs/News.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## October 9, 2019
+*  **Added data** consisting of  2615 samples from 3 studies:
+
```
Comment (jjgao) [inline docs/News.md]: How about https://www.cbioportal.org/study/summary?id=coad_cptac_2019? I am wondering if any other studies were missed?

Please add coad_cptac_2019 to the default set.

File: docs/News.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## October 9, 2019
+*  **Added data** consisting of  2615 samples from 3 studies:
+   
+     * [Cancer Cell Line Encyclopedia (Broad, 2019)](https://www.cbioportal.org/study/summary?id=ccle_broad_2019) *1739 samples*
```
Comment (jjgao) [inline docs/News.md]: please replace the 2012 study with the 2019 study in the Cell lines section.

File: docs/News.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+    * [Chronic Lymphocytic Leukemia (Broad, Nature 2015)](https://www.cbioportal.org/study/summary?id=cll_broad_2015#summary) *537 samples*
```
Comment (jjgao) [inline docs/News.md]: Add cll_broad_2019 to default set.

File: docs/News.md (hunk 2 above)
Comment (ritikakundra) [reply docs/News.md]: @jj niki suggested keeping both and but add a note with the link to the new one

File: docs/News.md (hunk 1 above)
Comment (ritikakundra) [reply docs/News.md]: done

File: docs/News.md (hunk 2 above)
Comment (jjgao) [reply docs/News.md]: sounds good

**Pull request merged**